snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/AbstractIntegrationTest.java

```diff
@@ -80,10 +80,10 @@ public static void setup() throws IOException {
                 inactive_count = 27820;
                 expected_supercs_cnt = 606769;
             } else if (absolutePath.contains("20250301")) {
-                stated_count = 400647;
```

Comment:
Just wondering - Would it be possible to embed these 'magic numbers' within some metadata/properties file in the SpinedArrayStore?

Reply:
We have had discussions in the past about the best way to track expected counts. Tracking them within the metadata of the generated database is an option that has been discussed for the future. For now, putting these counts in the abstract test class functions well until we get to a "better" solution.